protos/bottle/fulfillment/v1/line_item.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineItem {
+  string id = 1;
+  int32 quantity = 2;
+
+  bottle.catalog.v1.Product catalog_product = 3;
+
+  bool rush = 6;
+  int64 rush_price = 7;
```

Comment:
At first glance, this doesn't feel right. Rush would apply to 1 of 2 things: rush shipping, rush assembly but as an attribute on _every_ line item especially with a price? This just feels like the wrong place for this information.

What's the thinking with this living here?

Reply:
Currently we only have one type of rush. Rush assembly. Rush shipping is just a different type of shipping. It's also attached to the order product line currently. I could change the `rush` from a bool to an enum instead.

Reply:
Removed from the line item and included as a `catalog_addon`